modules/local/combine_tsv.nf

```diff
@@ -0,0 +1,64 @@
+process COMBINE_TSV {
```

Comment:
Why is this process needed? Can we have one process that reads in both, id or quant mode tsv files, filters user-defined columns and creates additional mqc files? Or is there some sort of blocker that I did not think of?